hostinit/setup.go

```diff
@@ -504,6 +504,14 @@ func (init *HostInit) copyScript(ctx context.Context, target *host.Host, name, s
 		append([]string{"-P", hostInfo.Port}, sshOptions...))
 
 	if err = scpCmd.SetOutput(output); err != nil {
+		grip.Alert(message.WrapError(err, message.Fields{
+			"runner":    RunnerName,
+			"operation": "setting up copy script command",
+			"distro":    target.Distro.Id,
+			"host":      target.Host,
```

Comment:
target.Host.Id? not sure if that's what you meant

Reply:
this is the `host_id` from the database, and we use this field throughout